test/Models/Tab.spec.js

```diff
@@ -54,13 +57,18 @@ describe('Tab Attributes', () => {
     });
   });
   describe('Association: TabDrinks', () => {
-    test('Tab is associated with tabDrinks', () => {
-      expect(tab.tabDrinkId).toBeTruthy;
+    let tabDrink;
+    beforeAll(async () => {
+      const { tabDrinks } = await Tab.findByPk(tab.id, {
+        include: { model: TabDrink },
+      });
+      tabDrink = tabDrinks[0];
+    });
+    test('TabDrinks belong to a Drink', () => {
+      expect(tabDrink.drinkId).toBe(drink.id);
     });
-  });
-  describe('Association: TabDrinks', () => {
     test('TabDrinks belong to a Tab', () => {
-      expect(drink.tabId).toEqual(tab.id);
+      expect(tabDrink.tabId).toBe(tab.id);
```

Comment:
Neither the `Tab` nor `Drink` has an association with the other; instead the `TabDrink` through table has both a `tabId` and `drinkId` to map the two together for the associations.